packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3917,18 +3922,18 @@ The following Factor types are supported:
 
 | Factor Type           | Description                                                                                                         |
 | --------------------- | ------------------------------------------------------------------------------------------------------------------- |
-| `call`                | Software [OTP](http://en.wikipedia.org/wiki/One-time_password) sent using voice call to a registered phone number     |
-| `email`               | Software [OTP](http://en.wikipedia.org/wiki/One-time_password) sent using email                                       |
+| `call`                | Software one-time passcode (OTP) sent using voice call to a registered phone number     |
```

Comment:
Why are we removing this link `http://en.wikipedia.org/wiki/One-time_password`

Reply:
The convention on our site is to avoid using wikipedia links (it's not a reliable source and can be a liability).